codex-rs/core/src/codex.rs

```diff
@@ -549,6 +566,11 @@ async fn submission_loop(
 
         debug!(?sub, "Submission");
         match sub.op {
+            Op::EraseConversationHistory => {
```

Comment:
Wherever you move this variant in the `enum` definition is probably where it should appear in this `match` statement.